x-pack/plugins/apm/ftr_e2e/cypress/e2e/service_map/service_map.cy.ts

```diff
@@ -50,25 +50,35 @@ describe('service map', () => {
   describe('when navigating to service map', () => {
     beforeEach(() => {
       cy.intercept('GET', '/internal/apm/service-map?*').as('serviceMap');
-      cy.visitKibana(serviceMapHref);
-
-      cy.wait('@serviceMap');
     });
 
-    it('shows nodes in service map', { retries: 3 }, () => {
+    it('shows nodes in service map', () => {
+      cy.visitKibana(serviceMapHref);
+      cy.wait('@serviceMap');
       cy.wait(500);
-      cy.getByTestSubj('serviceMap').matchImage();
+      cy.getByTestSubj('serviceMap').matchImage({
+        imagesPath: '{spec_path}/snapshots',
+        matchAgainstPath: 'cypress/e2e/service_map/snapshots/service_map.png',
+        maxDiffThreshold: 0.05, // maximum threshold above which the test should fail
+      });
     });
 
     it('shows nodes in detailed service map', () => {
       cy.visitKibana(detailedServiceMap);
+      cy.wait('@serviceMap');
       cy.contains('h1', 'opbeans-java');
       cy.wait(500);
-      cy.getByTestSubj('serviceMap').matchImage();
+      cy.getByTestSubj('serviceMap').matchImage({
+        imagesPath: '{spec_path}/snapshots',
+        matchAgainstPath:
+          'cypress/e2e/service_map/snapshots/detailed_service_map.png',
```

Comment:
Is this needed? Afaict it will use the name of the test as the image title.

Reply:
It was creating file names that are not allowed. So I force checking with images stored with allowed file names
<img width="836" alt="image" src="https://github.com/elastic/kibana/assets/31922082/979d9a85-0bdc-4664-b518-9a77e0d63515">

Reply:
All right. If you want, that rule can be disabled for the snapshots. That would be ok.

Alternatively you can specify the image name using `title` - you don't need to specify the full path: 
```suggestion
        title: 'my_screenshot',
```

Can you look into moving these settings to the config file instead if they are the same for every `matchImage` command?

Reply:
I tried that and was still adding the `#0`

Reply:
@sqren here you can see what I was talking about the title is still adding a  #0 even if you set it different https://github.com/FRSOURCE/cypress-plugin-visual-regression-diff/discussions/88#discussioncomment-3818468

Reply:
And here: https://github.com/FRSOURCE/cypress-plugin-visual-regression-diff/discussions/189#discussion-4590549